client/src/components/HomePage/ContactUs/index.js

```diff
@@ -9,38 +9,46 @@ const ContactUS = () => (
         <h1 className="titleTitle">CONTACT US</h1>
         <a name="Contact" title="slider" />
         <div className="contactInnerContainer">
-        <div className="contactUs">
-        <input
-            type="text"
-            placeholder="Enter Your Email"
-            className="inputField"
-          />
-          <input
-            type="text"
-            placeholder="Enter Message Title"
-            className="inputField"
-          />
-          <textarea
-            placeholder="Enter Your Message"
-            className="inputField textArea"
-          />
-          <button type="button" className="contactButton">
-            Send
-          </button>
-        </div>
-        <div className="contactUs">
-          <div className="contact-div">
-            <img src="https://imgur.com/vPSLDju.png" alt="icon"/>
-            <h3>+999-9999-9999</h3>
-          </div>
-          <div className="contact-div">
-            <img src="https://imgur.com/tkLsAgy.png" alt="icon"/>
-            <h3>sara-cahriZone@gmail.com</h3>
-          </div>
-          <div className="contact-div">
-            <img src="https://imgur.com/hahEuHs.png" alt="icon"/>
-            <h3>@ChariZone</h3>
+          <div className="contactUs">
+            <form
+              action="https://formspree.io/example@example.com"
```

Comment:
@othman-shamla 
Here, it might be helpful to leave yourself a `TODO` comment, to help yourself remember to fill in a real email address for this form 😀